automata/fa/dfa.py

```diff
@@ -313,16 +399,20 @@ def read_input_stepwise(
         if not ignore_rejection:
             self._check_for_input_rejection(current_state)
 
+    # Supports partial
     def _get_digraph(self) -> nx.DiGraph:
         """Return a digraph corresponding to this DFA with transition symbols ignored"""
-        return nx.DiGraph(
+        graph = nx.DiGraph(
             [
                 (start_state, end_state)
                 for start_state, transition in self.transitions.items()
                 for end_state in transition.values()
             ]
         )
+        graph.update(nodes=(state for state in self.states))
```

Comment:
```suggestion
        graph.update(self.states)
```
Can this just take in any iterable?

Reply:
Updated here https://github.com/caleb531/automata/pull/147/commits/7123f7ec24d8ec0fe1555e1c61aeb213880142d3